### PR TITLE
feat(cloud): add new repo

### PR DESCRIPTION
**Pull request by jasonkuhrt**

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+  "editor.codeActionsOnSave": {
+    "source.organizeImports": false
```
Comment (jasonkuhrt) [inline .vscode/settings.json:3]: I have this turned on in my user settings. But it was messing with the nice P1/P2 import order in main.ts.

File: src/common/labels.ts [hunk 2]
```diff
@@ -0,0 +1,198 @@
+import { Label } from '../labelsync-wrapper'
```
Comment (jasonkuhrt) [inline src/common/labels.ts:1]: This module is ported over from Prisma Labs. We used it to establish some patterns.

File: src/labelsync-wrapper.ts [hunk 3]
```diff
@@ -0,0 +1,51 @@
+import {
```
Comment (jasonkuhrt) [inline src/labelsync-wrapper.ts:1]: This module is taken from Prisma Labs. We found some annoyances with the labelsync API that we rectified here.

One issue was that we couldn't specify repo names in a modular way.

Another issue was that classes were API overkill.

File: src/main.ts [hunk 4]
```diff
@@ -1,64 +1,66 @@
...
-    "prisma2-private": prisma2Private,
-    "prisma-engines": prismaEngines,
-    "language-tools": languageTools,
-    "prisma-examples": prismaExamples,
-    "prisma-client-go": prismaClientGo,
+    'prisma2-private': prisma2Private,
+    'prisma-engines': prismaEngines,
+    'language-tools': languageTools,
+    'prisma-examples': prismaExamples,
+    'prisma-client-go': prismaClientGo,
```
Comment (jasonkuhrt) [inline src/main.ts:51]: automatic prettier run...

File: src/common/labels.ts (hunk 2 above)
Comment (janpio) [reply src/common/labels.ts:1]: probably name differently as it is a different "set" of defaults?
cloud-labels? labs-labels? or another way to "name" this set of defaults?

**Comment by jasonkuhrt** [comment]
@janpio I would be happy to introduce a refactor PR where we move everything to the labelsync wrapper api. Would not take long. And we'll know from the github bot that the refactor does not actually change any existing labels, so no risk it seems to me.

**Review by janpio — APPROVED** [review]
Looks good besides that one filename

File: src/common/labels.ts (hunk 2 above)
Comment (jasonkuhrt) [reply src/common/labels.ts:1]: `cloud-labels` sounds good for now!

Comment (jasonkuhrt) [reply src/common/labels.ts:1]: Actually gave me an idea to further isolate repos+common labels to a cloud dir.

**Pull request merged**